test/tests/functional/pbs_multi_sched.py

```diff
@@ -1644,34 +1645,42 @@ def test_auto_sched_off_due_to_fds_limit(self):
         when number of open files per process are exhausted
         """
 
-        # get the number of open files per process
-        ret = self.du.run_cmd(
-            self.server.hostname, [
-                'ulimit', '-n'], as_script=True, logerr=False)
-        self.assertEqual(ret['rc'], 0)
-        open_files_limit = ret['out'][0]
+        if os.getuid() != 0 or sys.platform in ('cygwin', 'win32'):
+            self.skipTest("Test need to run as root")
```

Comment:
While I'm fine with this, I do know that PDO does not run their tests as root.  You might consider consulting with them.